src/features/scheduled-messages.ts

```diff
@@ -36,43 +118,39 @@ export const scheduleMessages = (
   messageConfigs: MessageConfig[],
 ) => {
   const scheduledTasks = messageConfigs.map((messageConfig) =>
-    scheduleMessage(bot, messageConfig),
+    scheduleTask(messageConfig.interval, () => {
```

Comment:
I briefly tried, but the most obvious way produced a promise that wouldn't resolve for multiple hours which seemed bad. And since it's a recurring task, a single promise resolution doesn't really make sense